README.md

```diff
@@ -1 +1,37 @@
-# code-runner
\ No newline at end of file
+Code Runner: An Environment to run your code
+===================================================
+Code Runner is a web application to run code snippet or code files on a docker environment with code specific configurations.
+
+This application is mainly designed for running students' assignments for different courses.
+
+## Features
+- It has two ends for users - 
+    - One end from where professors can create course specific docker image.
+    - Second where students can submit code and view the output.
+- Currently it supports only server features - 
+    - Run the server
+    - Accept code files in compressed format
+    - Accept optional command line argument required to run code
+- Allows .zip, .tar and .tar.gz compression formats.
```

Comment:
Maybe put the _supported file types_ in a sub-section of its own?

Reply:
Done